src/nullable_format.rs

```diff
@@ -640,6 +717,7 @@ where
 mod serde_tests {
     use super::*;
     use serde::{Deserialize, Serialize};
+    use serde_json::json;
```

Comment:
```suggestion
    #[cfg(feature = "serdevalid")]
    use serde_json::json;
```

Reply:
Actually, I'd get rid of this entirely, and just fully qualify the single usage below.

```suggestion
```